multicluster/service-mirror/cluster_watcher.go

```diff
@@ -673,22 +681,32 @@ func (rcsw *RemoteClusterServiceWatcher) Stop(cleanupState bool) {
 	rcsw.eventsQueue.ShutDown()
 }
 
-func (rcsw *RemoteClusterServiceWatcher) resolveGatewayAddress() []corev1.EndpointAddress {
+func (rcsw *RemoteClusterServiceWatcher) resolveGatewayAddress() ([]corev1.EndpointAddress, error) {
 	var gatewayEndpoints []corev1.EndpointAddress
+	var errors []error
 	for _, addr := range strings.Split(rcsw.link.GatewayAddress, ",") {
-		resolved := addr
 		ipAddr, err := net.ResolveIPAddr("ip", addr)
 		if err == nil {
-			resolved = ipAddr.String()
+			gatewayEndpoints = append(gatewayEndpoints, corev1.EndpointAddress{
+				IP: ipAddr.String(),
+			})
+		} else {
+			errors = append(errors, fmt.Errorf("Error resolving '%s': %s", addr, err))
```

Comment:
Do we want to print a warning if some of the addresses resolve but others do not?

Reply:
Yep that might prove to be useful :+1: